character.py

```diff
@@ -1 +1,16 @@
+def __init__character(name, status, dialogue, thirst, hunger, injuryTimer, sicknessTimer):
```

Comment:
great start!! just missing the methods - but I can see some comments outlining some method ideas! both would work!!